app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.split()[1] != command.split()[2]:
+        with (open(
+                f"{command.split()[1]}",
+                "r"
+        ) as first_file,
+            open(
+                f"{command.split()[2]}",
+                "w"
```

Comment:
you split the same line 4 times. If it's a big string, it'll take some time, let's save splitted command in a variable and reuse it ;)